### ISSUE TITLE
Commands do not build windows paths correctly

### ISSUE THREAD

**Issue by bendem** [wslutilities/wslu#133]
**Background Information:**
 - Build: 
```
➜  ~ wslview  --version
/bin/wslview: line 108: /mntc/Windows/System32/reg.exe: No such file or directory
wslu v2.3.6; wslview v06
```
 - WSL: 
```
➜  ~ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.1 LTS"
```
 - Version: See above

**Describe the bug**
Binaries seem to drop a slash from the mountpath of disks, resulting in no tool under windows to be found:
```
➜  ~ wslfetch
/bin/wslfetch: line 108: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 108: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 183: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 185: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 187: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 189: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 108: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 183: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 185: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 187: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 189: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 108: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 183: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 185: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 187: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 189: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 108: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 183: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 185: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 187: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 189: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 108: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 183: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 185: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 187: /mntc/Windows/System32/reg.exe: No such file or directory
/bin/wslsys: line 189: /mntc/Windows/System32/reg.exe: No such file or directory

               .-/+oossssoo+/-.               Windows 10 Linux Subsystem
           `:+ssssssssssssssssss+:`           bendem@CL1277
         -+ssssssssssssssssssyyssss+-         BUILD:
       .ossssssssssssssssssdMMMNysssso.       BRANCH:
      /ssssssssssshdmmNNmmyNMMMMhssssss/      RELEASE:  Ubuntu 20.04.1 LTS
     +ssssssssshmydMMMMMMMNddddyssssssss+     KERNEL:   Linux 4.4.0-18362-Microsoft
    /sssssssshNMMMyhhyyyyhmNMMMNhssssssss/    UPTIME:   1d 5h 59m
   .ssssssssdMMMNhsssssssssshNMMMdssssssss.
   +sssshhhyNMMNyssssssssssssyNMMMysssssss+
   ossyNMMMNyMMhsssssssssssssshmmmhssssssso
   ossyNMMMNyMMhsssssssssssssshmmmhssssssso
   +sssshhhyNMMNyssssssssssssyNMMMysssssss+
   .ssssssssdMMMNhsssssssssshNMMMdssssssss.
    /sssssssshNMMMyhhyyyyhdNMMMNhssssssss/
     +sssssssssdmydMMMMMMMMddddyssssssss+
      /ssssssssssshdmNNNNmyNMMMMhssssss/
       .ossssssssssssssssssdMMMNysssso.
         -+sssssssssssssssssyyyssss+-
           `:+ssssssssssssssssss+:`
               .-/+oossssoo+/-.

```

**To Reproduce**
Run any command, see error

**Expected behavior**
A clear and concise description of what you expected to happen:
No error

**Additional context**

```
➜  ~ cat /etc/wsl.conf
[automount]
root = /mnt
options = "metadata,umask=22,fmask=11"
```

**Comment by patrick330602** [comment]
can you change the "root = /mnt" to "root = /mnt/" in your /etc/wsl.conf for now to temporarily solve the issue? The fix for this issue will be included in the 3.2.0 backport to `wslu` in Ubuntu.

**Comment by patrick330602** [comment]
Although this is not a Ubuntu-specific bug, you have to follow this bug to track the progress for `wslu` for Ubuntu: https://bugs.launchpad.net/ubuntu/+source/wslu/+bug/1888869, keep in mind closing this bug does not mean it is closed for Ubuntu.
